spec/models/mixins/drift_state_mixin_spec.rb

```diff
@@ -0,0 +1,100 @@
+describe DriftStateMixin do
+  include Spec::Support::ArelHelper
+
+  let(:host) { FactoryGirl.create(:host) }
+
+  let(:drift_states) do
```

Comment:
Minor, but if this is a `let!` then you don't need to call it in any of the below methods for "setup"

Reply:
Oh, nvm...I didn't see that one test expressly didn't need it...

That being said, perhaps there can be a `context "with some drift states"` around all of the tests that need that setup?

Again, it's minor...won't let it hold up a merge.